doc/devguide/codebase/code-style.rst

```diff
@@ -6,12 +6,165 @@ Suricata uses a fairly strict coding style. This document describes it.
 Formatting
 ~~~~~~~~~~
 
+clang-format
+^^^^^^^^^^^^
+
+.. Argh, github does not support admonitions such as .. note::
```

Comment:
The general idea behind these changes here are:

- mention usage of clang-format / git-clang-format to format changes only
- update the code style guide
- mention the corresponding clang-format setting

Does this make it a bit overloaded? Is this workable?

Reply:
I think it makes sense.